tests/TestCase.php

```diff
@@ -19,6 +20,8 @@ protected function setUp(): void
         $this->usesTestingFeature(new WithMigration('laravel', 'queue'));
 
         parent::setUp();
+
+        AliasLoader::getInstance()->setAliases([]);
```

Comment:
This ensures we don't rely on aliases across Pulse to help detect this in the future via the testsuite.